contracts/modules/utils/MultiCallUtils.sol

```diff
@@ -0,0 +1,95 @@
+pragma solidity 0.7.4;
+pragma experimental ABIEncoderV2;
+
+import "../commons/interfaces/IModuleCalls.sol";
+
+
+contract MultiCallUtils {
+  function multiCall(
+    IModuleCalls.Transaction[] memory _txs
+  ) public returns (
+    bool[] memory _successes,
+    bytes[] memory _results
+  ) {
+    _successes = new bool[](_txs.length);
+    _results = new bytes[](_txs.length);
+
+    for (uint256 i = 0; i < _txs.length; i++) {
+      IModuleCalls.Transaction memory transaction = _txs[i];
+
+      require(!transaction.delegateCall, 'MultiCallUtils#multiCall: delegateCall not allowed');
+      require(gasleft() >= transaction.gasLimit, "MultiCallUtils#multiCall: NOT_ENOUGH_GAS");
+
+      // solhint-disable
+      (_successes[i], _results[i]) = transaction.target.call{
+        value: transaction.value,
+        gas: transaction.gasLimit == 0 ? gasleft() : transaction.gasLimit
+      }(transaction.data);
+      // solhint-enable
+
+      require(_successes[i] || !_txs[i].revertOnError, 'MultiCallUtils#multiCall: CALL_REVERTED');
```

Comment:
What's the use case to use revertOnError? I guess could allow to return answer earlier in some cases?